internal/controller/integrationpipeline/integrationpipeline_adapter.go

```diff
@@ -90,6 +91,14 @@ func (a *Adapter) EnsureStatusReportedInSnapshot() (controller.OperationResult,
 			return err
 		}
 
+		integrationPipelineRunURL := status.FormatPipelineURL(a.pipelineRun.Name, a.pipelineRun.Namespace, a.logger.Logger)
+		if integrationPipelineRunURL == "" {
+			return fmt.Errorf("failed to get integration pipeline run console URL")
```

Comment:
I'm thinking that this might not be worth throwing an error for - maybe we can do something like this instead:
```
if integrationPipelineRunURL != "" {
    err = a.annotateIntegrationPipelineRunLogURL(a.context, a.client, a.pipelineRun, tekton.PipelinesAsCodePrefix+"/log-url", integrationPipelineRunURL)
	 if err != nil {
	     return err
	}
}
		
```